docs/examples/README.md

```diff
@@ -1,17 +1,14 @@
 # Examples
 
-This directory contains examples of DALI usage. List of examples:
+This directory contains examples of DALI usage.
 
 - [Getting started](getting%20started.ipynb) - build your first pipeline with DALI
-- [Augmentation gallery](augmentation_gallery.ipynb) - showcases different augmentations
-- [Color space conversion](color_space_conversion.ipynb) - showcases different color space conversion operations (e.g. RGB to YCbCr)
-- [LMDB data loading](dataloading_lmdb.ipynb) - example of reading data from LMDB database
-- [TFRecord data loading](dataloading_tfrecord.ipynb) - example of reading data from TensorFlow's TFRecord file
-- [RecordIO data loading](dataloading_recordio.ipynb) - example of reading data from MXNet's RecordIO file
-- [Serialization](serialization.ipynb) - example of pipeline serialization
+- [General](general) - General topics about using DALI, regardless if you want to do image, audio or any other processing.
+- [Image Processing](image_processing), [Sequence Processing](sequence_processing), [Audio Processing](audio_processing) - Tutorials, that present topics specific to the type of processed data
+- [Use Cases](use_cases) - Take a look on some real-life scenarios, where DALI can proove to be useful.
```

Comment:
I think this is an artificial category that tries to group examples form different categories.

Reply:
We can put there, e.g. pipelines for SSD, audio pipelines used in some models, real-life scenarios, this kind of stuff.

Reply:
pipelines for SSD -> mage Processing
audio pipelines -> Audio Processing
Still it doesn't make sense to put them in to one bag when we have this fine grind separation somewhere else.